core/src/main/java/by/ilyin/core/util/email/EmailProcessService.java

```diff
@@ -0,0 +1,53 @@
+package by.ilyin.core.util.email;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.stereotype.Component;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.MimeMessage;
+import java.io.File;
+
+@Component
+@RequiredArgsConstructor
+public class EmailProcessService {
+
+    private final JavaMailSender javaMailSender;
```

Comment:
@YauhenIlyin Are you serious?
Did you test it????
Where is configured JavaMailSender bean in your PR?

Reply:
it is built based on the configuration in the application file.

since the mail service may change or there may be several of them, and the password may also change, I considered it logical to remove the configuration from the code.

Reply:
@YauhenIlyin pls show me where credential are in application file?

Reply:
mail:
    host: smtp.gmail.com
    port: 587
    username: ilyin.testemail@gmail.com
    password: qftjaebtvnakoqff
    properties:
      mail:
        smtp:
          auth: true
          starttls:
            enable: true
        debug : true

Reply:
i tested it, everything worked